src/QuickMarcEditor/QuickMarcEditorRows/LinkButton/LinkButton.js

```diff
@@ -33,13 +33,14 @@ const LinkButton = ({
   handleUnlinkAuthority,
   isLinked,
   tag,
+  marcRecordId,
 }) => {
   const intl = useIntl();
   const [authority, setAuthority] = useState(null);
   const [isModalOpen, setIsModalOpen] = useState(false);
   const callout = useCallout();
 
-  const { isLoading, refetch: refetchSource } = useMarcSource(authority?.id, {
+  const { isLoading, refetch: refetchSource } = useMarcSource(marcRecordId, authority?.id, {

```

Comment:
Let's rename `marcRecordId` to `fieldId`. `marcRecordId` makes me think that we're passing Bib record's id and that's not the case
Otherwise, looks good!

Reply:
Thanks, corrected.